### DISCUSSION TITLE
How to make my own built app connect with my server

### DISCUSSION THREAD

**Discussion by devashish-gopalani-cognoai** [RocketChat/Rocket.Chat.ReactNative#3174] [Q&A] [answered]
We have setup an instance of rocketchat on our server. We are able to successfully use the web version of the rocket chat app. We have also taken a pull of this repo and added our own firebase push notification service so that we can get notifications on our android devices. The only doubt which I had was that how could I link my app which I have built with the server on which we have setup our rocketchat instance?

**Comment by diegolmello** [comment, marked as answer]
https://developer.rocket.chat/mobile-app/mobile-app-white-labelling
